.github/workflows/build.yml

```diff
@@ -1,13 +1,44 @@
 name: GitHub CI
 on: push
 
+env:
+  dotnet_version: '3.0.100'
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
```

Comment:
I prefer to not touch this env var in CI builds

Reply:
Sure thing